app/messages/inbound/send-notification.js

```diff
@@ -36,10 +36,10 @@ const sendNotification = async (message) => {
 
     try {
       if (response) {
-        await logCreatedNotification(message, response.data.id)
+        await logCreatedNotification(message, emailAddress, response.data.id)
       } else {
         await publishStatus(message, notifyStatus.INTERNAL_FAILURE, notifyError.response.data)
```

Comment:
Have updated the the call on `publishStatus` to include the `emailAddress` argument. Test has also been updated and everything is passing. Regarding the `check-notify-status/index.js` file, not 100% sure what change is needed since the `recipient` is already passed as one of the arguments for `publishStatus` and for `buildUpdateMessage`? Let me know and will make the update, thanks. 👍🏽

Reply:
Have made the updates 😄 This is now ready for re-review. Thanks.